_inc/lib/core-api/class.jetpack-core-api-site-endpoints.php

```diff
@@ -60,4 +66,160 @@ public static function get_features() {
 	public static function can_request() {
 		return current_user_can( 'jetpack_manage_modules' );
 	}
+
+
+	/**
+	 * Gets an array of data that show how Jetpack is currently being used to benefit the site.
+	 *
+	 * @since 7.7
+	 *
+	 * @return WP_REST_Response
+	 */
+	public static function get_benefits() {
+		$benefits = array();
+
+		/*
+		 * We get different benefits from Stats:
+		 * - this year's visitors
+		 * - Followers (only if subs module is active)
+		 * - Sharing counts (not currently supported in Jetpack -- https://github.com/Automattic/jetpack/issues/844 )
+		 */
+		$stats = null;
+		if ( function_exists( 'stats_get_from_restapi' ) ) {
+			$stats = stats_get_from_restapi( array( 'fields' => 'stats' ) );
+		}
+
+		// Yearly visitors.
+		if ( null !== $stats && $stats->stats->visitors > 0 ) {
+			$benefits[] = array(
+				'name'        => 'jetpack-stats',
+				'title'       => esc_html__( 'Jetpack Stats', 'jetpack' ),
+				'description' => esc_html__( 'Visitors tracked by Jetpack this year', 'jetpack' ),
+				'value'       => absint( $stats->stats->visitors ),
+			);
+		}
+
+		// Protect blocked logins.
+		if ( Jetpack::is_module_active( 'protect' ) ) {
+			$protect = get_site_option( 'jetpack_protect_blocked_attempts' );
+			if ( $protect > 0 ) {
+				$benefits[] = array(
+					'name'        => 'protect',
+					'title'       => esc_html__( 'Brute force protection', 'jetpack' ),
+					'description' => esc_html__( 'The number of malicious login attempts blocked by Jetpack', 'jetpack' ),
+					'value'       => absint( $protect ),
+				);
+			}
+		}
+
+		// Number of followers.
+		if ( null !== $stats && $stats->stats->followers_blog > 0 && Jetpack::is_module_active( 'subscriptions' ) ) {
+			$benefits[] = array(
+				'name'        => 'subscribers',
+				'title'       => esc_html__( 'Subscribers', 'jetpack' ),
+				'description' => esc_html__( 'People subscribed to your updates through Jetpack', 'jetpack' ),
+				'value'       => absint( $stats->stats->followers_blog ),
+			);
+		}
+
+		// VaultPress backups.
+		if ( Jetpack::is_plugin_active( 'vaultpress/vaultpress.php' ) && class_exists( 'VaultPress' ) ) {
+			$vaultpress = new VaultPress();
+			if ( $vaultpress->is_registered() ) {
+				$data = json_decode( base64_decode( $vaultpress->contact_service( 'plugin_data' ) ) ); // phpcs:ignore WordPress.PHP.DiscouragedPHPFunctions.obfuscation_base64_decode
+				if ( $data->features->backups && $data->backups->stats->revisions > 0 ) {
+					$benefits[] = array(
+						'name'        => 'jetpack-backup',
+						'title'       => esc_html__( 'Jetpack Backup', 'jetpack' ),
```

Comment:
Should it be Jetpack Backup**s**?

Reply:
I don't think so based on how we did the title for [this landing page](https://jetpack.com/upgrade/backup/), but I've found it both ways looking around the code base. I'll check with Rob first thing Monday when he's back from AFK regarding the copy for these and then merge the PR.